tests/calphy/test_base.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import os
+import numpy as np
+import unittest
+import warnings
+
+from pyiron_atomistics.project import Project
+from pyiron_atomistics.calphy.calphy import Calphy
+from pyiron_base import state, ProjectHDFio
+
+class TestCalphy(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        state.update({'resource_paths': os.path.join(os.path.dirname(os.path.abspath(__file__)), "../static")})
+        cls.execution_path = os.path.dirname(os.path.abspath(__file__))
+        #cls.file_location = os.path.dirname(os.path.abspath(__file__))
+        cls.project = Project(os.path.join(cls.execution_path, "test_calphy"))
+        cls.job = Calphy(
+            project=ProjectHDFio(project=cls.project, file_name="test_calphy"),
+            job_name="test_calphy",
+        )
+        filepath = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../static/")
+        cls.output_project = Project(os.path.join(filepath, "calphy_test_files"))
+        cls.output_job = Calphy(
+            project=ProjectHDFio(project=cls.output_project, file_name="tm_fcc"),
+            job_name="tm_fcc",
+        )
+        #cls.job = cls.project.create_job("Calphy", "job_calphy")
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.execution_path = os.path.dirname(os.path.abspath(__file__))
+        project = Project(os.path.join(cls.execution_path, "test_calphy"))
+        project.remove_jobs_silently(recursive=True)
+        project.remove(enable=True)
+        state.update()
```

Comment:
```suggestion
    @classmethod
    def tearDownClass(cls):
        cls.execution_path = os.path.dirname(os.path.abspath(__file__))
        project = Project(os.path.join(cls.execution_path, "test_calphy"))
        project.remove_jobs_silently(recursive=True)
        project.remove(enable=True)
        state.update()
```
You'll want to derive from `pyiron_base._tests.TestWithProject`.